src/sentry/api/endpoints/organization_stats_v2.py

```diff
@@ -0,0 +1,225 @@
+from rest_framework.response import Response
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.utils import get_date_range_rollup_from_params
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.models import Project, Team
+
+from sentry.snuba import outcomes
+from sentry_relay import DataCategory
+from sentry.utils.outcomes import Outcome
+from collections import defaultdict
+import collections.abc
+
+from sentry.utils.snuba import (
+    naiveify_datetime,
+    to_naive_timestamp,
+)
+from copy import deepcopy
+
+
+CATEGORY_NAME_MAP = {
+    DataCategory.ERROR: "statsErrors",
+    DataCategory.TRANSACTION: "statsTransactions",
+    DataCategory.ATTACHMENT: "statsAttachments",
+}
+
+DEFAULT_TS_VAL = {
+    "accepted": {"quantity": 0, "times_seen": 0},
+    "filtered": {"quantity": 0, "times_seen": 0},
+    "dropped": {
+        "overQuota": {"quantity": 0, "times_seen": 0},
+        "spikeProtection": {"quantity": 0, "times_seen": 0},
+        "other": {"quantity": 0, "times_seen": 0},
+    },
+}
+
+
+class OrganizationStatsEndpointV2(OrganizationEndpoint):
+    def get(self, request, organization):
+        start, end, rollup = get_date_range_rollup_from_params(request.GET, "1h", round_range=True)
+        result = outcomes.query(
+            start=start,
+            end=end,
+            rollup=rollup,
+            groupby=["category", "time", "outcome", "reason"],
+            aggregations=[["sum", "times_seen", "times_seen"], ["sum", "quantity", "quantity"]],
+            filter_keys={"org_id": [organization.id]},
+            orderby=["time"],
+        )
+
+        response = {
+            "statsErrors": defaultdict(lambda: deepcopy(DEFAULT_TS_VAL)),
+            "statsTransactions": defaultdict(lambda: deepcopy(DEFAULT_TS_VAL)),
+            "statsAttachments": defaultdict(lambda: deepcopy(DEFAULT_TS_VAL)),
```

Comment:
Nit: instead of the constant `DEFAULT_TS_VAL`, defining

```python
def create_default_ts_value():
    return {...}
```

would let you just write `defaultdict(create_default_ts_value)`, avoiding the use of both `lambda` and `deepcopy`.